File: Python/screen_info.py
def screen(employee):
    """Creates a string containing an employee's employment information.

    Args:
        employee (Employee): employees.Employee object describing the employee

    Returns:
        str: A string containing the employee's name, employment type, number of
            years with the company, and accrued vacation time
    """
    output = 'Name: {} {}, ' \
             'Duration: {} years, ' \
             'Vacation Accrued: {}{}'.format(employee.name,
                                             employee.type,
                                             employee.longevity,
                                             employee.vacation,
                                             ' days' if employee.vacation is not
                                             None else '')
    return output


def screen_list(employees_list):
    """Prints employment data for a list of employees.
    Args:
        employees_list (List[Employee]): A list of employees.Employee objects
            describing the employees whose data should be printed
    """
    for employee in employees_list:
        print screen(employee)

File: C++/ScreenInfo.cpp
#include <iostream>
#include <string>
#include <vector>
#include "Employees.h"

std::string screen(Employee employee)
{
     std::string s;

     s = "Name: " + employee.getName() + " " + employee.getType() +
          ", Duration: " + std::to_string(employee.getLongevity()) +
          " years, Vacation Accrued: ";
     if (employee.getVacation() != NULL)
     {
          s += std::to_string(employee.getVacation());
          s += " days";
     }
     else
     {
          s += "None";
     }

     return s;
}

void screen_list(std::vector<Employee> employees)
{
     for (int i = 0; i < (employees.size() / sizeof(Employee)); i++)
     {
          std::cout << screen(employees[i]);
     }
}